simpa/core/device_digital_twins/pa_devices/ithera_msot_acuity.py

```diff
@@ -190,7 +188,7 @@ def update_settings_for_use_of_model_based_volume_creator(self, global_settings)
         self.set_detection_geometry(detection_geometry)
         for illumination_geom in self.illumination_geometries:
             illumination_geom.device_position_mm = np.add(illumination_geom.device_position_mm,
-                                                          np.array([0, 0, probe_size_mm]))
+                                                          np.array([width_shift_for_structures_mm, 0, probe_size_mm]))
```

Comment:
I do not understand where this is coming from exactly.. Is it for "sliding" a probe along the x-axis to potentially simulate multiple areas of the same tissue model?